instrumentation/apache-camel-2.20/src/main/java/io/opentelemetry/javaagent/instrumentation/apachecamel/decorators/KafkaSpanDecorator.java

```diff
@@ -61,22 +61,22 @@ public void pre(Span span, Exchange exchange, Endpoint endpoint, CamelDirection
 
     String partition = getValue(exchange, PARTITION, Integer.class);
     if (partition != null) {
-      span.setAttribute("partition", partition);
+      span.setAttribute("apache-camel.kafka.partition", partition);
```

Comment:
What do you think about using `SemanticAttributes.MESSAGING_KAFKA_PARTITION` for this one?